drivers/gpio/Kconfig.pcf8574

```
@@ -0,0 +1,19 @@
+# PCF8574 GPIO configuration options
+
+# Copyright (c) 2022 Ithinx
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig GPIO_PCF8574
+	bool "PCF8574 I2C GPIO chip"
+	default y
+	depends on DT_HAS_NXP_PCF8574_ENABLED
+	depends on I2C
```

The current policy is to `select` the bus.
```suggestion
	select I2C
```